main_keysync.go

```diff
@@ -59,6 +65,10 @@ func main() {
 		"(optional) config file for keyprotect enablement")
 	flag.StringVar(&inputFlags.keyprotectConfigKubeSecret, "keyprotectConfigKubeSecret", inputFlags.keyprotectConfigKubeSecret,
 		"(optional) kube secret name for config file for keyprotect enablement")
+	flag.StringVar(&inputFlags.keyFilePermissions, "keyFilePermissions", inputFlags.keyFilePermissions,
+		"(optional) permissions for the created key files (defaults to 0600)")
+	flag.StringVar(&inputFlags.keyFileOwnership, "keyFileOwnership", inputFlags.keyFileOwnership,
+		"(optional) ownership for the created key files (in uid:gid format)")
```

Comment:
add text that "default that it will create in the UID/GID of the process", since its not clear what `-1:-1` means.